python/kserve/kserve/logging.py

```diff
@@ -0,0 +1,51 @@
+# Copyright 2022 The KServe Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+import logging.config
+
+from .constants.constants import KSERVE_LOGLEVEL
+
+KSERVE_LOGGER_NAME = 'kserve'
+KSERVE_LOGGER_FORMAT = ('%(asctime)s.%(msecs)03d %(process)s %(name)s '
+                        '%(levelname)s [%(funcName)s():%(lineno)s] %(message)s')
+KSERVE_LOGGER_DATE_FORMAT = "%Y-%m-%d %H:%M:%S"
+
+logging.config.dictConfig({
+    'version': 1,
+    'disable_existing_loggers': False,
```

Comment:
Thanks @cmaddalozzo for making the code change! Can we please add `'disable_existing_loggers': False,` to the rest server logger [here](https://github.com/kserve/kserve/blob/ab71c1c3767c9d1e81e873702e5e46f29a57c65c/python/kserve/kserve/protocol/rest/server.py#L140) as well? In my understanding the `rest_server` might still disable all other loggers without `'disable_existing_loggers': False,`

Reply:
Added!